### PR TITLE
Formplayer request

### PR DESCRIPTION
**Pull request by wpride**
Toggle to use Formplayer, also include domain in Touchforms requests (needed for toggle)

cross: https://github.com/dimagi/touchforms/pull/199

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -565,6 +565,8 @@
 FORMPLAYER_URL = 'http://localhost:8080'
 OFFLINE_TOUCHFORMS_PORT = 4444
 
+USE_FORMPLAYER = False
```
Comment (benrudolph) [inline settings.py]: why can't we use the toggle you setup here? https://github.com/dimagi/commcare-hq/pull/9999/files#diff-57f91d9d96727cedf8225cb4c7d50b74R633

**Comment by wpride** [comment]
@benrudolph yep, forgot to push

cross https://github.com/dimagi/touchforms/pull/199

**Comment by benrudolph** [comment]
test failure looks real

**Comment by wpride** [comment]
man I suck at subrepo commits

**Pull request merged**